lib/octopus_serverspec_extensions/matcher/use_guided_failure.rb

```diff
@@ -0,0 +1,13 @@
+RSpec::Matchers.define :use_guided_failure do
```

Comment:
what do these matchers do that the type doesn't?

Reply:
Let's see if I can word this correctly... There are some common matcher verbs that spec automatically 'translates' (for want of a better word) into a method call. A method on the type ends up looking a bit clumsy. example:

`should be_in_space -calls-> .in_space?`
`.has_description? -> expect(thing).to have_description`

It won't do that kind of thing with "use" unless you go down the matcher route. Which gives you both worlds.